packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -223,14 +245,14 @@ export const BasicMultiCombobox: React.FC = () => {
             type="text"
             {...getInputProps(getDropdownProps({preventKeyAction: isOpen}))}
             {...getToggleButtonProps({tabIndex: 0})}
-            value={selectedItem || ''}
           />
         </Box>
         {isOpen && (
           <ComboboxListbox {...getMenuProps()}>
             <ComboboxListboxGroup>
               {filteredItems.map((item, index) => (
                 <ComboboxListboxOption
+                  variant="default"
```

Comment:
Added variant b/c got typescript error that variant is required on ComboboxListboxOption